### PR TITLE
Moved search widget to be below view pager tabs.

### PR DESCRIPTION
**Pull request by genaforvena**
Replaced add art object stub floating button with search button.

Removed stub methods from data source.

### CODE REVIEW

**Comment by genaforvena** [comment]
@sanovikov71 
@oleg-popenov

**Comment by genaforvena** [comment]
Fixes #46

**Comment by genaforvena** [comment]
#64

File: app/src/main/java/org/imozerov/streetartview/location/LocationHelper.kt [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import android.location.LocationManager
+import android.util.Log
+import com.google.android.gms.maps.GoogleMap
+import com.google.android.gms.maps.model.LatLng
+import com.google.android.gms.maps.model.MarkerOptions
+
+/**
+ * Created by imozerov on 11.03.16.
+ */
+val DEFAULT_USER_LOCATION = LatLng(56.3298063, 44.0095144)
```
Comment (sanovikov71) [inline app/src/main/java/org/imozerov/streetartview/location/LocationHelper.kt:14]: What is it? Do we really need it?

Comment (genaforvena) [inline app/src/main/java/org/imozerov/streetartview/location/LocationHelper.kt:14]: Oh! No! We need it as we may get no user location and have to set camera somewhere etc.

File: app/src/main/java/org/imozerov/streetartview/storage/DataSource.kt [hunk 2]
```diff
@@ -53,52 +53,6 @@ class DataSource() : IDataSource {
...
-                name = "${randomFrom(firstArtPart)} ${randomFrom(lastArtPart)}"
-                id = Math.random().toString()
-                thumbPicUrl = "Pic"
-                picsUrls = RealmList<RealmString>()
-                location = realmLocation
-            }
-
-            it.insertOrUpdate(realmArtObject)
-        }
-    }
```
Comment (sanovikov71) [inline app/src/main/java/org/imozerov/streetartview/storage/DataSource.kt:101]: Well done!

File: app/src/main/java/org/imozerov/streetartview/ui/detail/GalleryPagerAdapter.kt [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+class GalleryPagerAdapter(val context: Context, val images: List<String>, listenerAction: ((position: Int) -> (Unit))) : PagerAdapter() {
+    val inflater: LayoutInflater
+    val listenerAction: ((Int) -> (Unit))
+
+    init {
+        inflater = context.getSystemService(Context.LAYOUT_INFLATER_SERVICE) as LayoutInflater
+        this.listenerAction = listenerAction
+    }
+
+    override fun getCount(): Int {
```
Comment (sanovikov71) [inline app/src/main/java/org/imozerov/streetartview/ui/detail/GalleryPagerAdapter.kt]: Make a oneliner  
override fun getCount() = images.size

**Pull request merged**